extension/logstash/lib/logstash/outputs/doris.rb

```diff
@@ -90,17 +90,17 @@ def print_plugin_info()
    end
 
    def register
-      @http_query = "/api/#{db}/#{table}/_stream_load"
+      @http_query = "/api/#{@db}/#{@table}/_stream_load"
```

Comment:
Is '@' necessary?

Reply:
It's not necessary, but it's usually added in logstash plugin.